components/oauth/org.wso2.carbon.identity.oauth.common/src/main/java/org/wso2/carbon/identity/oauth/common/OAuthConstants.java

```diff
@@ -165,4 +165,14 @@ private OAuthError(){
     private OAuthConstants(){
 
     }
+
+    public static final String ID_TOKEN_HINT = "&id_token_hint=";
+    public static final int STATUS_OK = 200;
+    public static final String MSG = "Invalid logout request";
+    public static final String RP_STATUS_LOGOUT = "True";
```

Comment:
Variable name and its value does not give any sense, better to use meaning full names